### DISCUSSION TITLE
Achievements

### DISCUSSION THREAD

**Discussion by meyoh-23** [community/community#82830] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

got a Beta badge 2 weeks ago, and I,m proud of it

**Comment by Talha-Hamza** [comment]
That's great! I am glad you did.

**Comment by ananuness** [comment]
How to get it and why?

**Reply by mw-xyz** [reply]
He's gotten the "Quickdraw" achievement, which is when you close a pull request within 5 minutes of opening. BETA just shows because he has unlocked an achievement.

**Comment by MEClouds** [comment]
The achievements listed below are ones that you can obtain:
| Name | Badge Image | Earnable? | Earned By | Difficulty |
| --- | --- | --- | --- | --- |
Pull Shark | <img src="https://raw.githubusercontent.com/Schweinepriester/github-profile-achievements/main/images/pull-shark-default.png" width="175px"> | `✅ Yes` | You can get this badge by merging 2 pull requests. | Medium
YOLO | <img src="https://raw.githubusercontent.com/Schweinepriester/github-profile-achievements/main/images/yolo-default.png" width="175px"> | `✅ Yes` | A user can obtain this badge by merging a pull request without a review (This requires 1 user and 1 reviewer) | Medium
Quickdraw | <img src="https://raw.githubusercontent.com/Schweinepriester/github-profile-achievements/main/images/quickdraw-default.png" width="175px"> | `✅ Yes` | Obtainable by closing a Pull Request or Issue in 5 mins of it being opened. (You can close your own issues) | Very Easy
Pair Extraordinaire	 | <img src="https://raw.githubusercontent.com/Schweinepriester/github-profile-achievements/main/images/pair-extraordinaire-default.png" width="175px"> | `✅ Yes` | The user can obtain this badge by [Coauthoring](https://docs.github.com/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors) in a merged pull request. (Requires GitHub Desktop App) | Hard
Galaxy Brain | <img src="https://raw.githubusercontent.com/Schweinepriester/github-profile-achievements/main/images/galaxy-brain-default.png" width="175px"> | `✅ Yes` | A user can get this badge by getting 2 of their answers accepted on the [Community Discussions](https://github.com/orgs/community/discussions/) forum. | Medium
Starstruck | <img src="https://raw.githubusercontent.com/Schweinepriester/github-profile-achievements/main/images/starstruck-default.png" width="175px"> | `✅ Yes` | This badge is given to someone when one of their repository's gets 16 stars. | Hard
Public Sponsor | <img src="https://raw.githubusercontent.com/Schweinepriester/github-profile-achievements/main/images/public-sponsor-default.png" width="175px"> | `✅ Yes` | This badge can be earned by sponsoring a repository/organization/user. | Very Easy

**Comment by Talha-Hamza** [comment]
To get you started, here is how you can obtain one of the hard badges above, the **Pair Extraordinaire** badge : 

https://github.com/Prakash4844/Get-Pair-Extraordinaire/tree/main

**Comment by github-staff** [comment]
Thanks so much for posting in our Community @meyoh-23. Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your  ability to earn badges in other projects and communities across GitHub? This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Comment by diobenu2silva** [comment]
I can exchange stars. Please help each other.

https://github.com/diobenu2silva/pumpfun_SC
https://github.com/diobenu2silva/zeroGpt4
https://github.com/diobenu2silva/solArmy_staking_SC
https://github.com/diobenu2silva/triangular_arbitrage_multi_exchange

**Reply by codershubinc** [reply]
@diobenu2silva 
let's exchange stars
 [star this open-api](https://github.com/codershubinc/open-api)
 ## I stared your repo [star this open-api](https://github.com/codershubinc/open-api)
![Screenshot_2025-07-01-21-49-33-73_320a9a695de7cdce83ed5281148d6f19](https://github.com/user-attachments/assets/03532e8b-1919-4c74-8a83-44b3d9548dc8)

---
this is my account also.
